R/data_loading_helpers.R

```diff
@@ -148,3 +148,297 @@ collect_relative_risks <- function(in_list, keyword_map) {
     return(out)
   }, simplify = FALSE, USE.NAMES = TRUE)
 }
+
+get_directory_names <- function(dir_name, return_paths = FALSE) {
```

Comment:
I don't see any difference in the output when using `return_paths = TRUE` vs `FALSE`, but maybe I don't understand it correctly

Reply:
The printed output is the same because the full paths are returned only as an invisible character vector. So if you would do `paths <- get_risk_factor_names(root_dir, return_paths = TRUE)`, the `paths` object would contain the full paths. This is also mentioned in the documentation.